src/components/MainSection.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import Components from '/images/image3.png';
+import Singleway from '/images/image4.png';
+import JSX from '/images/image5.png';
+
+function MainSection() {
+  return (
+    <section className="main-section">
+      <div className="main-section__container">
+        <div className="main-section__row">
```

Comment:
why not make this a new component like "card" receive props image tittle and text section like this 

```
<Card title=' ' image={} text=''/>

```